CONTRIBUTING.md

```diff
@@ -45,6 +45,9 @@ issues.
 Please note that the minimum supported version of Alacritty is Rust 1.41.0. All patches are expected
 to work with the minimum supported version.
 
+If you have made any breaking changes to `alacritty_terminal`, make sure to bump its version
```

Comment:
This part seems out of place, knowing that you have literally the same message later.

Reply:
I've considered the same, but this section is about contributions to Alacritty while the other section is about releases.

I think it's relevant in both sections, so I thought it makes sense to state it twice even if there is a slight overlap.

Reply:
It just sounds to me like. If you have made any breaking changes to `alacritty_terminal` from your patch, then update version according to semver as well. My main concern here is not that we have overlap, but because it's really confusing the way it is right now. It's pretty clear in `Release` section, since you're doing release, and since versions are synced, it tells you what to do, and how to bump. The `Release` applies even to contributors, don't you think?

Reply:
Say the current state of alacritty_terminal is 0.10.0-dev, then a release would bump it to 0.10.1-dev, since there obviously haven't been any breaking changes after 0.10.0 (there haven't been any changes at all!).

However, now a user writes a patch and sends a PR that changes the public interface of Alacritty. At that point keeping the version at 0.10.1-dev would be incorrect, since we cannot release it as 0.10.1-dev anymore. So the version needs to be bumped to 0.11.0.

The section at the top instructs contributors to bump the version according to semver, the section at the bottom instructs publishers to bump just the patch version after a release. They aren't the same and they have different purposes.

Reply:
>However, now a user writes a patch and sends a PR that changes the public interface of Alacritty. At that point keeping the version at 0.10.1-dev would be incorrect, since we cannot release it as 0.10.1-dev anymore. So the version needs to be bumped to 0.11.0.

Oh, that way, then you should add it to PR template as well?

Reply:
Probably a good idea.